lib/gossip/dissemination.js

```diff
@@ -22,13 +22,14 @@
 var EventEmitter = require('events').EventEmitter;
 var util = require('util');
 
+var EMPTY = [];
 var LOG_10 = Math.log(10);
 
 function Dissemination(ringpop) {
     this.ringpop = ringpop;
     this.ringpop.on('ringChanged', this.onRingChanged.bind(this));
 
-    this.changes = {};
+    this.membershipChanges = {};
```

Comment:
NOTES FOR REVIEWER: Renamed var to be more explicit about what type of changes are being disseminated.